mwdb/web/src/components/Navigation.js

```diff
@@ -180,6 +187,7 @@ function mapStateToProps(state, ownProps)
         ...ownProps,
         error: state.config.error,
         config: state.config.config,
+        remotes: state.config.config ? state.config.config["remotes"] : [],
```

Comment:
I would use `config` directly (`this.props.config.remotes.length`). Now it may look dirty, but it's more compatible with context-based version (in https://github.com/CERT-Polska/mwdb-core/pull/268)

Reply:
This implementation was used originally, but it caused an error (TypeError: this.props.config is null), so I decided to write the code this way. Of course, in the new version it will be changed, but implementing a different way at the moment crashes the UI.

Reply:
Ok, sounds reasonable.